sros2/test/sros2/commands/security/verbs/test_generate_policy.py

```diff
@@ -24,8 +24,6 @@
 from test_msgs.srv import Empty
 
 
-# TODO(ivanpauno): reactivate this test after updating generate policy
-@pytest.mark.skip(reason='temporarily deactivated')
 def test_generate_policy_topics():
```

Comment:
It might be nice to test this by setting a different enclave path to the created node. Not sure how to do this from `rclpy` and not `ros2 run <package> <node> -e <path>` CLI. @ivanpauno ?

Reply:
You should be able to pass commandline arguments to `rclpy.init`
https://github.com/ros2/rclpy/blob/f1a09c20a4af221da7db476206fea48a83284db7/rclpy/rclpy/__init__.py#L66

Reply:
see https://github.com/ros2/sros2/pull/203/commits/ff3503be530c7f488dffccaa33b0a62247761732